### ISSUE TITLE
train code and parameters

### ISSUE THREAD

**Issue by kjlee18** [daniegr/EfficientPose#36]
Hi, 
I am new to pose estimation task and interested in your paper.
On your repo, test code is good enough for me, however, no train codes.

My wish is to fine-tune your model on anime dataset to generate keypoint dataset for anime images.
I tried to reproduce train code referring your paper, but some hyper-parameters and loss are ambiguous to me.

1. In equation (4) and (5), what values are for initial and last gamma value?
2. Also, for training loss, did you use only sigmoid cross entropy loss on the probability of equation (4)? Or other losses needed?

Could you answer to my questions? If possible, could you also load the train code? It will be really helpful for me

Thanks

**Comment by daniegr** [comment]
@kjlee18: Thank you for your interest in EfficientPose! Unfortunately, we are not planning to launch the training code at the moment. Regarding the loss function, please have a look at #23.  For sigma values, see Fig. 6 for EfficientPose II. For other models, values are linearily scaled in relation to the resolution of detection passes. Moreover, in case of fine-tuning you may also simplify the process by utilizing the Adam optimizer rather than SGD.
